local-cluster/src/cluster_tests.rs

```diff
@@ -60,7 +60,7 @@ pub fn spend_and_verify_all_nodes<S: ::std::hash::BuildHasher + Sync + Send>(
             return;
         }
         let random_keypair = Keypair::new();
-        let client = create_client(ingress_node.client_facing_addr());
+        let client = create_client(ingress_node.client_facing_addr().0, ingress_node.client_facing_addr().1);
```

Comment:
Let's not call client_facing_addr or other functions to get contact info multiple times.

Reply:
Changed, thank you!